docs/architecture/sstable/sstable3/index.rst

```diff
@@ -12,21 +12,24 @@ Scylla SSTable - 3.x
 
 .. include:: ../_common/sstable_what_is.rst
 
-* In Scylla 3.1 and above, mc format is enabled by default. 
+* In ScyllaDB 5.1 and above, the ``me`` format is enabled by default (the ``enable_sstables_me_format`` option is set to ``true``).
```

Comment:
There's no `enable_sstables_me_format` in 5.1, and the `enable_sstables_m[cd]_format` were deprecated too.
Instead, starting 5.1 we can now say ```
the ``sstable_format`` option is set to ``me``
```

Reply:
@bhalevy I see, thanks. Is this information relevant from the user's perspective? Will they ever have to/want to set `sstable_format` to anything else than `me'? I'm asking because:
- If yes, we need to document the `sstable_format` option (I'll open a new issue to do that). Btw, the `enable_sstables_m[cd]_format` option was never documented.
- If no, it would be better to remove the option names and simply state "In ScyllaDB 5.1 and above, the ``me`` format is enabled by default."

Reply:
I've never see a user or field engineering actually use it.
I see this option as useful as "first aid" to customers that happen to stumble a particular bug that prevents them from using the latest format. (@vladzcloudius / @slivne have you ever seen those options used in the field?)

IMO it is good to document those options in general, but here it's ok to just just mention that the "me" format is enabled by default without going into details.

Reply:
@bhalevy Thanks! I've removed the option name as it seems to be redundant in that place in the documentaiton.